pkg/executor/brie.go

```diff
@@ -392,7 +388,9 @@ func (b *executorBuilder) buildBRIE(s *ast.BRIEStmt, schema *expression.Schema)
 		}
 
 	case ast.BRIEKindRestore:
-		e.restoreCfg = &task.RestoreConfig{Config: cfg}
+		rcfg := task.DefaultRestoreConfig()
+		rcfg.Config = cfg
```

Comment:
It seems that `rcfg.Config` has been parsed in the `task.DefaultRestoreConfig()`.

Reply:
We will override the default config with the new config. Will this make things wrong?